paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/SimpleTextHeaderElement.kt

```diff
@@ -0,0 +1,53 @@
+package com.stripe.android.paymentsheet.elements
+
+import android.content.res.Resources
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.paymentsheet.forms.FormFieldEntry
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import java.io.InputStream
+import java.util.Locale
+
+internal data class SimpleTextHeaderElement(
+    override val identifier: IdentifierSpec,
+    private val paymentMethodType: PaymentMethod.Type?,
+    override val controller: Controller? = null,
+) : FormElement() {
+    override fun getFormFieldValueFlow(): Flow<List<Pair<IdentifierSpec, FormFieldEntry>>> =
+        MutableStateFlow(emptyList())
+
+    fun getLabel(resources: Resources): String {
```

Comment:
Take a look at the MandateSpec and MandateELement.  I would recommend having this be a simpleTextHeaderElement and just passing in a string resource.   Then this doesn't have to know about Klarna.

Reply:
Spoke offline. We decided to modify mandate text and make it more generic for this.